modules/controller/permissioned-data-controller.js

```diff
@@ -1,16 +1,16 @@
-const BN = require('bn.js');
+const BN = require('../../node_modules/bn.js/lib/bn');
```

Comment:
Why load node modules like this?

Reply:
This must be because of the merge, you can change it to normal :-)